Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/git-repo.json

```diff
@@ -145,5 +159,67 @@
             "identifier": "2.2@branch-a", 
             "revision": 6
         }
+    ],
+    "eng/squash-branch": [
```

Comment:
I am fine adding a branch here, but it's worth noting that individual tests can add branches and commits as well.